components/program_manager/spec/lib/time_client_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+
+RSpec.describe TimeClient do
+  describe '.get_time' do
+    subject(:get_time) { described_class.time_at(timezone_name) }
+
+    context 'when timezone format is correct' do
+      let(:timezone_name) { 'America/Vancouver' }
+
+      before do
+        stub_request(:get, "http://worldtimeapi.org/api/timezone/America/Vancouver")
```

Comment:
I guess my idea up above wasn't necessary, about abstracting to a method, woops! 🤦🏽